### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by MohammadAlTurany**
Describe your proposal.

Mention any issue this PR resolves or is related to.

---

Checklist:

* [ ] Rebased against `dev` branch
* [ ] My name is in the resp. CONTRIBUTORS/AUTHORS file
* [ ] Followed [the seven rules of great commit messages](https://chris.beams.io/posts/git-commit/#seven-rules)

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
+Next-generation Particle Physics Experiments at [GSI](https://www.gsi.de)/[FAIR](https://www.gsi.de/forschungbeschleuniger/fair) and [CERN](https://home.web.cern.ch/) are facing [unprecedented data processing challenges](https://doi.org/10.1051/epjconf/201921405010). The expected data rates require a non-trivial amount of high performance compute (HPC) resources, i.e: Thausends of CPU/GPU cores per experiment. Online data processing/compression are crucial to fulfill the storage capacity requirements. The complexity of tasks that need to be performed during the online (synchronous) data processing is significantly higher than ever before. Complex tasks that usually runs offline (e.g: Calibration, Track finding, etc) have to run now online in a high performance and high throughput environment.
```
Comment (dennisklein) [inline readme.md:3]:
```suggestion
Next-generation Particle Physics Experiments at [GSI](https://www.gsi.de)/[FAIR](https://www.gsi.de/forschungbeschleuniger/fair) and [CERN](https://home.web.cern.ch/) are facing [unprecedented data processing challenges](https://doi.org/10.1051/epjconf/201921405010). The expected data rates require a non-trivial amount of high performance compute (HPC) resources, i.e. thousends of CPU/GPU cores per experiment. Online data processing/compression are crucial to fulfill the storage capacity requirements. The complexity of tasks that need to be performed during the online (synchronous) data processing is significantly higher than ever before. Complex tasks that usually runs offline (e.g: Calibration, Track finding, etc) have to run now online in a high performance and high throughput environment.
```

Comment (rbx) [reply readme.md:3]: thousands*

Comment (dennisklein) [inline readme.md:3]:
```suggestion
Next-generation Particle Physics Experiments at [GSI](https://www.gsi.de)/[FAIR](https://www.gsi.de/forschungbeschleuniger/fair) and [CERN](https://home.web.cern.ch/) are facing [unprecedented data processing challenges](https://doi.org/10.1051/epjconf/201921405010). The expected data rates require a non-trivial amount of high performance compute (HPC) resources, i.e: Thausends of CPU/GPU cores per experiment. Online data processing/compression are crucial to stay within storage capacity limits. The complexity of tasks that need to be performed during the online (synchronous) data processing is significantly higher than ever before. Complex tasks that usually runs offline (e.g: Calibration, Track finding, etc) have to run now online in a high performance and high throughput environment.
```

Comment (rbx) [reply readme.md:3]: I would also switch `have to run now online` to `have to now run online`.

**Comment by rbx** [comment]
Updated.

**Comment by rbx** [comment]
Updated again...

**Review by rbx — APPROVED** [review]

**Pull request merged**